bors/src/project_board.rs

```diff
@@ -261,7 +261,41 @@ impl ProjectBoard {
         Ok(())
     }
 
-    async fn list_cards(github: &GithubClient, column_id: u64) -> Result<Vec<ProjectCard>> {
+    // TODO: Merge with `list_cards`
+    pub async fn list_canary_cards(
+        &self,
+        ctx: &ActivePullRequestContext<'_>,
+    ) -> Result<Vec<ProjectCard>> {
+        let column_id = self.canary_column.id;
+        let github = ctx.github();
+        let mut list_options = ListProjectCardsOptions {
```

Comment:
this function seems just 
```
Self::list_cards(&github, self.canary_column.id)
```

Reply:
Yeah... I felt like I had that, and for some reason it didn't work, but now it does 🤷🏻‍♂️